matrix.yaml

```
@@ -9,22 +9,59 @@ PYTHON_VER:
   - "3.9"
   - "3.10"
 LINUX_VER:
+  - "ubuntu18.04"
   - "ubuntu20.04"
   - "ubuntu22.04"
   - "centos7"
   - "rockylinux8"
 IMAGE_REPO:
   - "ci"
+  - "ci-wheel"
+  - "citestwheel"
 exclude:
+  # ci:
   - LINUX_VER: "ubuntu22.04"
     CUDA_VER: "11.2.2"
   - LINUX_VER: "ubuntu22.04"
     CUDA_VER: "11.4.3"
   - LINUX_VER: "ubuntu22.04"
     CUDA_VER: "11.5.2"
+  # wheels:
```

```suggestion

  # wheels:
```

Since we're doing another round of changes, let's add a line break here for consistency with the rest of the excludes.